### PR TITLE
updating :matches() to :is(), as per spec change

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://github.com/w3c/csswg-drafts/issues/3258

### CODE REVIEW

File: css/selectors/is.json [hunk 1]
```diff
@@ -1,93 +1,99 @@
...
-          "description": "<code>:matches()</code>",
-          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:matches",
+          "description": "<code>:is()</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:is",
           "support": {
             "chrome": [
               {
-                "version_added": "66"
+                "version_added": "66",
+                "alternative_name": ":matches()"
```
Comment (ExE-Boss) [inline css/selectors/is.json]: This requires a flag.

File: css/selectors/is.json [hunk 2]
```diff
@@ -1,93 +1,99 @@
...
                 "version_added": "18",
-                "alternative_name": ":-webkit-any"
+                "alternative_name": ":-webkit-any()"
               }
             ],
             "edge": [
               {
-                "version_added": "15"
+                "version_added": "15",
+                "alternative_name": ":matches()"
```
Comment (ExE-Boss) [inline css/selectors/is.json]: ditto

**Comment by chrisdavidmills** [comment]
Happy to update this if Joe and Erika confirm the details ;-)

I can do some digging if needed, but I've run out of time for now.

**Comment by Elchi3** [comment]
This will conflict with https://github.com/mdn/browser-compat-data/pull/2937

**Comment by ExE-Boss** [comment]
Well then we should probably merge https://github.com/mdn/browser-compat-data/pull/2937 first (I&nbsp;had completely forgoten about its&nbsp;existence), as it corrects the data, and then rebase this on top of it.

**Comment by Elchi3** [comment]
@chrisdavidmills I'm sorry #2937 took so long. Can you rebase this PR now? Thank you!

**Comment by chrisdavidmills** [comment]
I can't work out how to rebase this correctly. I've just spent half an hour trying to do this and got myself in a complete mess. I'm just going to close this and make the changes in a new PR, which would've taken me 15 minutes if I'd just done that in the first place.

**Pull request closed without merging**